app/services/optout/OptOutProposition.scala

```diff
@@ -50,4 +50,6 @@ case class OptOutProposition(previousTaxYear: PreviousOptOutTaxYear,
     }
   }
 
+  val availableTaxYearsForOptOut: Seq[TaxYear] = availableOptOutYears.map(_.taxYear)
```

Comment:
lets define this variable before all functions

Reply:
Not sure what you mean please clarify?

Reply:
done